ModLoader.cs

```diff
@@ -127,37 +120,31 @@ public static void LoadFiles()
                 }
                 catch(Exception ex)
                 {
-                    Log.Information(ex, string.Format("Cannot read the mod {0}", file));
+                    Log.Information(ex, "Cannot read the mod {0}", file);
                 }
                 if (f == null) continue;
-                try
-                {
-                    Assembly assembly = f.Assembly;
-                    // for array or list, use the available search method instead of Linq one
-                    // use the Linq ones for IEnumerable
-                    Type? modType = Array.Find(assembly.GetTypes(), t => t.IsSubclassOf(typeof(Mod)));
 
-                    if (modType == null)
-                    {
-                        MessageBox.Show("加载错误: " + assembly.GetName().Name + " 此Mod需要一个Mod类");
-                        continue;
-                    }
-                    else
-                    {
-                        if (Activator.CreateInstance(modType) is not Mod mod) continue;
-                        mod.LoadAssembly();
-                        mod.ModFiles = f;
-                        f.instance = mod;
+                Assembly assembly = f.Assembly;
+                // for array or list, use the available search method instead of Linq one
+                // use the Linq ones for IEnumerable
+                Type? modType = Array.Find(assembly.GetTypes(), t => t.IsSubclassOf(typeof(Mod)));
+
+                if (modType == null)
+                {
+                    MessageBox.Show("加载错误: " + assembly.GetName().Name + " 此Mod需要一个Mod类");
```

Comment:
This can be translated to:

`MessageBox.Show("Loading error: " + assembly.GetName().Name + "This Mod need a Mod class");`